src/pages/open-source-for-non-developers.md

```diff
@@ -0,0 +1,31 @@
+# Open Source for Non-Developers
+
+## What is Open Source?
+
+Open source is software whose source code can be freely viewed and redistributed.
+
+Unlike products that contain proprietary code, open source code can be viewed by developers worldwide.
+
+It also allows people to propose changes, which fosters a collaborative working environment.
+
+## Why do Products Become Open Source?
+
+1. To interact with consumers: By making the designs and code open source, consumers have a more direct way to improve the products they enjoy,
+   which in turn can save your company time and effort.
+2. To reach more demographics: Open source products are often freely viewed by many consumers. This can help increase brand awareness.
+3. To build loyalty and trust: Since open source products allow people to see the source code. This creates transparency, which helps build trust between consumers and the product’s creators.
```

Comment:
```suggestion
3. To build loyalty and trust: Since open source products allow people to see the source code. 
This creates transparency, which helps build trust between consumers and the product’s creators.
```